tests/test_generate_index.py

```diff
@@ -135,9 +126,12 @@ def test_get_releases(setup_index_tests_target_branch):
     Path(f"target_worktree/{source_branch}/").mkdir(parents=True)
     open(f"target_worktree/{source_branch}/index.html", 'a').close()
     releases = get_releases(target_branch, True, source_branch, Path(f"target_worktree").absolute())
-    assert releases == [{'release': "latest", 'release_path': f'{source_branch}/index.html'},
+    correct_releases_local = [{'release': "latest", 'release_path': f'{source_branch}/index.html'},
                         {'release': 'feature-some-feature', 'release_path': 'feature-some-feature/index.html'},
                         {'release': 'another_branch', 'release_path': 'another_branch/index.html'}]
+    for release in releases:
+        assert release in correct_releases_local
+    assert len(releases) == len(correct_releases_local)
```

Comment:
Combine multiple asserts to one, because if you have multiple asserts the comparison stops after the first assert fails. The `release in correct_releases_local` you can translate to `set(releases) == set(correct_releases_local)`

Reply:
done, others in ticket